README.md

```diff
@@ -71,6 +71,8 @@ By default, the migration tool will load these values from a file named `vcs.jso
 Before initiating the migration process, first determine which command line arguments you wish to pass (if any).  The following arguments are currently supported:
 * `--vcs-file-path`: this flag allows you to pass a custom file path for your TFE_VCS_CONNECTION_MAP JSON file. If not specified, `vcs.json` will be used by default.
 * `--migrate-all-state`: this flag allows you to set the desired behavior for migrating state versions.  If passed, all versions of state will get migrated for all workspaces.  If not specified, only the current version of state for all workspaces will be migrated by default.
+* `--workspace-list-file-path`: this flag allows you to pass a custom file path for your SOURCE_WORKSPACE_LIST TXT file. If not specified, `workspace_list.txt` will be used by default.
+ * `--migrate-select-workspaces`: this flag allows you to set the desired behavior for migrating workspaces.  If passed, only the workspaces included in your SOURCE_WORKSPACE_LIST TXT file will be migrated.
```

Comment:
You'll need to update all the all the env vars to have `JSON` instead of `TXT`. Also missing an underscore before `TXT`.

Reply:
Done.